### PR TITLE
Add missing logs types to derived market db

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

File: packages/augur-sdk/src/state/db/DisputeDB.ts [hunk 1]
```diff
@@ -9,16 +9,16 @@ export class DisputeDatabase extends DerivedDB {
     protected processDoc(log: ParsedLog): ParsedLog {
         if (log.name === 'InitialReportSubmitted') {
             return this.processInitialReportSubmitted(log);
-        } else if (log.name === 'DisputeCrowdsourcerCompleted') {
-            return this.processDisputeCrowdsourcerCompleted(log);
+        } else if (log.name === 'DisputeCrowdsourcerCreated') {
+            return this.processDisputeCrowdsourcerCreated(log);
         } else if (log.name === 'InitialDisputeCrowdsourcerContribution') {
```
Comment (nuevoalex) [inline packages/augur-sdk/src/state/db/DisputeDB.ts]: This also needs to be changed to `DisputeCrowdsourcerContribution` and `processDisputeCrowdsourcerContribution`

File: packages/augur-sdk/src/state/db/MarketDB.ts [hunk 2]
```diff
@@ -47,6 +47,8 @@ export class MarketDB extends DerivedDB {
       'MarketVolumeChanged',
       'MarketOIChanged',
       'InitialReportSubmitted',
+      'DisputeCrowdsourcerCreated',
```
Comment (nuevoalex) [inline packages/augur-sdk/src/state/db/MarketDB.ts]: I think we actually dont need these

**Review by nuevoalex — APPROVED** [review]
Can remove the events added to MarketDB I believe but otherwise yay!

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 👏

 This link will always point to this PR's latest changes: https://peek.run/7g6hggg

**Comment by justinbarry** [comment]
~~Moved to #2778.~~

**Pull request merged**